pkg/token/keywords.go

```diff
@@ -79,6 +79,7 @@ var Keywords = []TokenKind{
 	"RECURSIVE",
 	"RESPECT",
 	"RIGHT",
+	"ROLE",
```

Comment:
Is `ROLE` actually keyword? At least, it is not listed in [official doc](https://cloud.google.com/spanner/docs/reference/standard-sql/lexical#reserved_keywords).

Reply:
Thanks for your review.
I removed `ROLE` from keywords.